app/templates/partials/header.jade

```diff
@@ -1,2 +1,16 @@
-header
-  p Header goes here
+nav(class="navbar navbar-default navbar-fixed-top")
+  div(class="container")
+    div(class="navbar-left")
+      div(class="navbar-header")
+        a(class="navbar-brand" href="#") AtMeBro
```

Comment:
multiple attributes in jade should be separated by a comma
`a(class="navbar-brand", href="#") AtMeBro`